backend/src/main/java/ai/verta/modeldb/project/FutureProjectDAO.java

```diff
@@ -146,4 +158,9 @@ public FutureProjectDAO(Executor executor, FutureJdbi jdbi, UAC uac) {
             },
             executor);
   }
+
+  public InternalFuture<Project> getPreAccessProjectById(String projectId) {
```

Comment:
What is this? Why is this public? Assuming `preAccess` means without authorization check, it doesn't make sense to be public, as per our guidelines https://github.com/VertaAI/docs/tree/main/engineering-practices/api-implementation

Reply:
removed code for now in PR: #2129